modules/controls.js

```diff
@@ -86,9 +86,16 @@ displaySystem.registerModule({
 
         function open() {
             if (config.url) {
+                //we have a completely custom control window supplied by the user, do not bother rendering
                 var win = window.open(config.url,'fllDisplayControlWindow','resize=yes,width=800,height=550');
             } else {
                 var win = window.open('','fllDisplayControlWindow','resize=yes,width=800,height=550');
+                if (win.document.body.innerHTML) {
```

Comment:
I think because of this test, the behavior is a bit inconsistent when switching between custom and standard control window. Test: start displaySystem, press c to open default controls, change config to simple custom control, refresh the main page, press c to open the console again and you get the new. Disable the url in the config, refresh main window, press c and console is not updated. Not sure if you can test if window.location.pathname = config.url?